### PR TITLE
Remove no-schema config

### PR DESCRIPTION
**Pull request by sunzhaoyang**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@sunzhaoyang Please invite a tech review. Thanks~

**Review by glorv — APPROVED** [review]
LGTM

File: migrate-from-csv-files-to-tidb.md [hunk 1]
```diff
@@ -32,22 +32,10 @@ CSV 文件自身未包含表结构信息。要将 CSV 数据导入 TiDB，就必
...
         - 文件名格式为 `${db_name}.${table_name}-schema.sql`，其内容需包含 `CREATE TABLE` 语句。
 
-    2. 后续导入过程中，在 `tidb-lightning.toml` 中添加如下设置：
-
-        ```toml
-        [mydumper]
-        no-schema = false # 通过 Lightning 在下游创建库和表，此项设为 false。
         ```
 
 * **方法二**：手动在下游 TiDB 建库和表。
```
Comment (hfxsd) [inline migrate-from-csv-files-to-tidb.md:37]:
```suggestion
```
方法二已经没有了。得删掉这句话。
第 L26、L28 也得更新下。我没法加 comments。请自行修改。

File: migrate-from-sql-files-to-tidb.md [hunk 2]
```diff
@@ -32,20 +32,8 @@ aliases: ['/docs-cn/dev/migrate-from-mysql-mydumper-files/','/zh/tidb/dev/migrat
 
     2. 后续导入过程中，在 `tidb-lightning.toml` 中添加如下设置：
 
-        ```toml
-        [mydumper]
-        no-schema = false # 通过 Lightning 在下游创建库和表，此项设为 false。
-        ```
-
 * **方法二**：手动在下游 TiDB 建库和表。
```
Comment (hfxsd) [inline migrate-from-sql-files-to-tidb.md:35]:
```suggestion
```
方法二已经没有了。得删掉这句话。
第 L24、L26 也得更新下。我没法加 comments。请自行修改。

**Review by hfxsd — APPROVED** [review]
Rest LGTM

File: migrate-from-csv-files-to-tidb.md (hunk 1 above)
Comment (sunzhaoyang) [reply migrate-from-csv-files-to-tidb.md:37]: 手动建就是方法二呀 ^_^

Comment (hfxsd) [reply migrate-from-csv-files-to-tidb.md:37]: 好吧，我还以为底下删掉的内容就是方法二。

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**